### PR TITLE
Add check for project master title to Cypress test

### PR DESCRIPTION
**Pull request by petterwildhagen**

### CODE REVIEW

File: frontend/src/views/Project/Dashboard/Components/TablesAndTitles.tsx [hunk 1]
```diff
@@ -10,9 +10,14 @@ const TablesAndTitles = ({ evaluationsWithProjectMasterTitle }: Props) => {
     return (
         <>
             {Object.entries(evaluationsWithProjectMasterTitle).map(([projectMasterTitle, evaluations], index) => {
+                const titleNoSpaces = projectMasterTitle.replace(/ /g, '')
```
Comment (maiaht) [inline frontend/src/views/Project/Dashboard/Components/TablesAndTitles.tsx:13]: What is replaced here? And why?

File: frontend/cypress/integration/landing_page_spec.ts [hunk 2]
```diff
@@ -176,33 +176,43 @@ describe('Landing page', () => {
...
                 })
             })
-            let portfolios = ['No portfolio']
-            projectMasters.forEach(pm => portfolios.push(pm.portfolioOrganizationalUnit))
-            it('Non hidden evaluations are listed under their respective portfolios', () => {
+
+            let portfolios = [{ portfolio: 'No portfolio', projectMasterTitle: 'No project master title' }]
+            projectMasters.forEach(pm => portfolios.push({ portfolio: pm.portfolioOrganizationalUnit, projectMasterTitle: pm.title }))
+            it.only('Non hidden evaluations are listed under their respective portfolios', () => {
+                console.log(projectMasters)
```
Comment (maiaht) [inline frontend/cypress/integration/landing_page_spec.ts:202]: Remove `console.log()`

File: frontend/cypress/integration/landing_page_spec.ts [hunk 3]
```diff
@@ -176,33 +176,43 @@ describe('Landing page', () => {
...
                     })
                 })
             })
-            let portfolios = ['No portfolio']
-            projectMasters.forEach(pm => portfolios.push(pm.portfolioOrganizationalUnit))
-            it('Non hidden evaluations are listed under their respective portfolios', () => {
+
+            let portfolios = [{ portfolio: 'No portfolio', projectMasterTitle: 'No project master title' }]
+            projectMasters.forEach(pm => portfolios.push({ portfolio: pm.portfolioOrganizationalUnit, projectMasterTitle: pm.title }))
+            it.only('Non hidden evaluations are listed under their respective portfolios', () => {
```
Comment (achaikou) [inline frontend/cypress/integration/landing_page_spec.ts:201]: And remove it.only

File: frontend/src/views/Project/Dashboard/Components/TablesAndTitles.tsx (hunk 1 above)
Comment (petterwildhagen) [reply frontend/src/views/Project/Dashboard/Components/TablesAndTitles.tsx:13]: Spaces are removed. Cypress cannot find an element with spaces in the name. I have removed the local variable and applied replace directly where attribute is set.

File: frontend/cypress/integration/landing_page_spec.ts (hunk 2 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/landing_page_spec.ts:202]: Deleted.

File: frontend/cypress/integration/landing_page_spec.ts (hunk 3 above)
Comment (petterwildhagen) [reply frontend/cypress/integration/landing_page_spec.ts:201]: Removed.

**Review by maiaht — APPROVED** [review]

**Pull request merged**